modules/mobile/app/controllers/mobile/v0/community_care_eligibility_controller.rb

```diff
@@ -4,6 +4,8 @@ module Mobile
   module V0
     class CommunityCareEligibilityController < ApplicationController
       def show
+        # i don't believe this is currently in use. this doesn't seem useful though because it doesn't
+        # reveal any information we can't get by looking at the endpoint logs
         Rails.logger.info('CC eligibility service call start',
```

Comment:
Category: I am here with supporting info.
Action: remove. it could be useful to log this on failure but isn't helpful on success. The service type is also a query param and should be visible in DD logs.